src/response.ts

```diff
@@ -27,10 +63,25 @@ document.addEventListener('DOMContentLoaded', async () => {
     // eslint-disable-next-line functional/immutable-data
     (el as HTMLElement).innerText = useremail;
   }
+
+  await getDataFromSessionStorageTask('payment')
+    .chain(transaction => checkStatusTask(transaction.token, paymentManagerClient))
+    .chain(transactionStatusResponse => isNot3dsFlowTask(transactionStatusResponse))
+    .fold(
+      _ => showErrorStatus(), // 3ds case
+      transactionStatusResponse =>
+        checkStatusTask(
+          Buffer.from(transactionStatusResponse.data.idTransaction.toString()).toString('base64'),
```

Comment:
@infantesimone isn't this the `token` field of `payment` data strucure in session storage?

Reply:
Yes it's the same, but it recomputed based on the transaction response, is it better to use the same?